madgraph.sh

```diff
@@ -10,7 +10,41 @@ build_requires:
 ---
 #!/bin/bash -e
 
-rsync -a --exclude='**/.git' --delete --delete-excluded $SOURCEDIR/ $INSTALLROOT/
+rsync -a --exclude='**/.git' --delete --delete-excluded $SOURCEDIR/ $BUILDDIR/
+
+# install internal packages 
+cd $BUILDDIR
+cat << EOF >> $PWD/install.dat
+install oneloop
+install ninja
+install collier
+install RunningCoupling
+install QCDLoop
+install MadAnalysis5
+EOF
+
+./bin/mg5_aMC $PWD/install.dat
+
+# cleanup after build
+rm  $PWD/install.dat
+rm HEPTools/ninja/ninja_install.log 
+rm HEPTools/oneloop/oneloop_install.log
+rm HEPTools/collier/collier_install.log
+rm HEPTools/madanalysis5/madanalysis5_install.log
+rm -rf HEPTools/ninja/Ninja 
+rm -rf HEPTools/oneloop/OneLOop*
+rm -rf HEPTools/collier/COLLIER*
+cd QCDLoop
+todelete=($(ls -1 | grep -v include | grep -v lib | grep -v share))
+for d in ${todelete[@]}; do rm -rf $d; done
```

Comment:
Instead of these three lines, I'd suggest:
```bash
find QCDLoop -mindepth 1 -maxdepth 1 -not -name include -not -name lib -not -name share -exec rm -rf {} +
```
That avoids parsing the output of `ls` and all the string-splitting hairness that comes with that.

Reply:
OK, this I can change